middleware.go

```diff
@@ -125,15 +129,25 @@ func Middleware(config Config) echo.MiddlewareFunc {
 				}
 			}
 
-			stop := time.Now()
+			// lookup response again in case it was modified
+			res = c.Response()
```

Comment:
this was required, otherwise the response looked up above still had the old value in it.